### PR TITLE
Make `pixelshuffle` scriptable

### LINKED ISSUE #4108: Make `pixelshuffle` scriptable.
**Is your feature request related to a problem? Please describe.**

Modify the `pixelshuffle` method in such a way that when using this upsampling method on any network, it can be scripted using `torch.jit.script`

**Describe the solution you'd like**

This should not raise an error:
```python
import torch
from monai.networks.utils import pixelshuffle

torch.jit.script(pixelshuffle)
```

### PR DESCRIPTION
**Pull request by ramonemiliani93**
Signed-off-by: Ramon Emiliani <ramon.emiliani@umontreal.ca>

Fixes #4108.

### Description
Modifies the existing `pixelshuffle` function in such a way that it is scriptable using `torch.jit.script`.

### Status
**Work in progress**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/networks/utils.py [hunk 1]
```diff
@@ -281,14 +281,16 @@ def pixelshuffle(
             f"divisible by scale_factor ** dimensions ({factor}**{dim}={scale_divisor})."
         )
 
-    org_channels = channels // scale_divisor
+    org_channels = int(channels // scale_divisor)
```
Comment (wyli) [inline monai/networks/utils.py:284]: thanks, could you help add a test case like this one https://github.com/Project-MONAI/MONAI/blob/1516ca758090dd8ed2890b64554a77f681a0e224/tests/test_se_blocks.py#L68-L72 for the subpixel upsampling https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_subpixel_upsample.py

Comment (ramonemiliani93) [reply monai/networks/utils.py:284]: Done @wyli 👌

**Comment by ramonemiliani93** [comment]
@wyli Seems `range` is not scriptable on PyTorch 1.7 and the test fails, what do you think we should do in this case?

**Comment by wyli** [comment]
> @wyli Seems `range` is not scriptable on PyTorch 1.7 and the test fails, what do you think we should do in this case?

was going to mention this, I added the condition to skip https://github.com/Project-MONAI/MONAI/pull/4109/commits/026f840a531be6749216b0fc62c87045826c7331

**Review by wyli — APPROVED** [review]
Thanks, it looks good to me.

**Pull request merged**